frontend/homework/1/index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <main>
+        <div class="webpage">
+
+            <div class="aboutme">
+
+        <header>
+            <div class="about">
+                <div id="name">
+                    <h2><u><i>Raghav Nandwana</i></u></h2>
+                </div>
+                <div id="photo">
+                    <img src="logan.jpg" alt="A picture of Raghav Nandwana">
+                    <aside>
+                        <p>An Image</p>
+                    </aside>
+                </div>
+            </div>
+
+        </header>
+
+        <div class="information">
+            <p>This is A Small Information Site About me. </p>
+            <h3>Hobbies</h3>
+            <ol>
+                <li>BasketBall</li>
+                <li>Computer Gaming</li>
+                <li>Cricket</li>
+            </ol>
+            <h3>Professional Profiles</h3>
+            <ul>
+                <li>GeeksForGeeks
+                    <a href="geeksforgeeks.com" target="_blank">
+                        <button>GeeksForGeeks</button>
+                    </a>
+                </li>
+                <li>GitHub
+                    <a href="Github.com" target="_blank">
+                        <button>GitHub</button>
+                    </a>
+                </li>
+
+            </ul>
+        </div>
+        </div>
+        <!-- this whole logic below asks the user 
+    to register using the forms in 
+    html -->
+
+        <div class="register">
+            <form action="">
+                <!-- The div contains the form  -->
+                
+                    <div class="register">
+                        <h3>
+                            <pre>Welcome : </pre>Register to my News Letter Here
+                        </h3>
+                        <p>Please fill these Details</p>
+                        <div class="form">
+                        
+                        <hr>
+                        <section id="email">
+                            <label for="email"><b>Email</b></label>
+                            <input type="text" placeholder="Enter Email" name="email" id="email" required>
+                        </section>
+
+                        <section id="password">
+                            <label for="password"><b>Password</b></label>
+                            <input type="password" placeholder="Enter Password" name="password" id="password" required>
+                        </section>
```

Comment:
This is not the right way to wrap each field into different sections
Consider section as a module of chapter